### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by xii-vi**
### Login
mobile (19)

### SignUp
RE-Verbnation (1)

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/pages/auth/login.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        e.preventDefault();
+        try{
+        const loginResponse = await getLoginDataFromServer(email,password);
+        console.log(loginResponse)
+        localStorage.setItem("encodedToken", loginResponse.data.encodedToken)
+        localStorage.setItem('userData', JSON.stringify(loginResponse.data.foundUser));
+        authDispatch({ type: "USER_LOGIN" })
+        authDispatch({ type: "USER_TOKEN", payload: loginResponse.data.encodedToken })
+        authDispatch({ type: "USER_DATA", payload: loginResponse.data.foundUser })
+        navigate("/")
```
Comment (TejasShekar) [inline src/pages/auth/login.jsx:28]: You can navigate programmatically to the previous page that user was on before user was redirected to login page

**Review by TejasShekar — APPROVED** [review]
Good File structure and code. 
You can merge.

File: src/pages/auth/login.jsx (hunk 1 above)
Comment (xii-vi) [reply src/pages/auth/login.jsx:28]: Sure i will add that to the code

**Pull request merged**